include/libtorrent/create_torrent.hpp

```diff
@@ -347,7 +347,7 @@ namespace libtorrent
 		// and they have the same name.
 		bool m_multifile:1;
 
-		// this is true if the torrent is private. i.e., is should not
+		// this is true if the torrent is private. i.e., its interested peers should not
```

Comment:
the typo here is that "is" should have been "it". I think it make more sense to specify whether *this* client will announce to the DHT or not. It doesn't really have control over what other peers do. also, "interested" is used in a specific meaning in the peer protocol, is that what you refer to?